resources/views/home.blade.php

```diff
@@ -2,6 +2,16 @@
 
 @section('content')
     <div class="main py-4">
+        @can("admin.versioncheck.read")
```

Comment:
i think this would be much better suited for an overview page, rather than the home page. but just a note.

Reply:
i thought it would be better suited on the homepage because this way an admin would instantly see it everytime he visits his own hosts site